metapool/rescale_counts.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import numpy as np
+
+
+def _to_column_percentage(df):
+    return df / df.sum() * 100
+
+
+def _to_row_percentage(df):
+    return df.div(df.sum(axis=1), axis=0) * 100
+
+
+def _to_log10_cpm(df):
+    aligned_reads = df.sum()
+    df_log10_cpm = np.log10(df / aligned_reads * 1000000)
+    return df_log10_cpm
+
+
+def _apply_model(df_log10_cpm, linear_models):
+    cols = []
+    failed_colnames = []
+    for c in df_log10_cpm:
+        r = linear_models[linear_models["sample_name"] == c]
+        if r.empty:
+            # TODO FIXME HACK:  What do we do if this sample can't be rescaled
```

Comment:
I think this is a really good question that we would need to address when it actually happens (so we can figure out why it happened). Could you raise an error so we are forced to take a look when it happens? Thank you.

Reply:
It happens in the test code in 2 of the samples and will cause the current tests to fail, as the R code also fails to calculate on these samples.